packages/proxy/schema/index.ts

```diff
@@ -49,10 +49,79 @@ export const modelParamToModelParam: {
   stream_options: null,
   parallel_tool_calls: null,
   response_format: null,
-  reasoning_effort: null,
+  reasoning_effort: "reasoning_effort",
   stop: null,
 };
 
+const effortToBudgetMultiplier = {
+  low: 0.2,
+  medium: 0.5,
+  high: 0.8,
+} as const;
+
+const getBudgetMultiplier = (effort: keyof typeof effortToBudgetMultiplier) => {
```

Comment:
tests 🙏🏾

Reply:
huge thanks for calling this one out. would have been 🔥 

tl;dr

the order that we call the mappers mattered!